src/shared/containers/MapTest.js

```diff
@@ -0,0 +1,103 @@
+import React, {PropTypes, Component} from 'react';
+import controllable from 'react-controllables';
+
+import GoogleMap from 'google-map-react';
+import Marker from '../components/Marker.js';
+
+class MapTest extends Component {
+  constructor(props) {
+    super(props);  
+
+    this.onBoundsChange = (center, zoom) => {
```

Comment:
As I commented in the other PR, this can be out of this constructor, could be a member function (method) of the class.